page/page.njk

```diff
@@ -1,5 +1,6 @@
 {% extends "template.njk" %}
 {%- from "moj/components/banner/macro.njk" import mojBanner -%}
+{%- from "moj/components/header/macro.njk" import mojHeader -%}
```

Comment:
is this being used? `mojHeader` is not being referenced, but I can see `govukHeader` being referenced below

Reply:
also. It advises not to use this for the public here: https://design-patterns.service.justice.gov.uk/components/header/